docs/ql-documentation/learn-ql/cpp/introduce-libraries-cpp.rst

```diff
@@ -4,114 +4,686 @@ Introducing the C/C++ libraries
+| Declaration syntax                                                                                                                                                                                                                                                                                                                                                                                                                          | QL class                                                                                                                                                              | Remarks                                                                                                                                                                                                                                                                           |
++=============================================================================================================================================================================================================================================================================================================================================================================================================================================+=======================================================================================================================================================================+===================================================================================================================================================================================================================================================================================+
+| ``enum`` ``{`` **enum_constant1** ``,`` **enum_constant2** ``,`` **enum_constant3** ``}``                                                                                                                                                                                                                                                                                                                                                   | `EnumConstant <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/Enum.qll/type.Enum$EnumConstant.html>`__                                                             |                                                                                                                                                                                                                                                                                   |
```

Comment:
Why are certain words such as "enum_constant1" and (later) "strcat" bold?

Reply:
Because the formatting has still not been formalized.  Which of the various "styles" contained in the document are preferable to you?

Reply:
I think I prefer when the declaration syntax is all in a monospaced font, with no bold or italics.  Bolding the element in question (as in `strcat`) also looks reasonable to me.  Consistency is important though.

Reply:
Be aware that unfortunately rst does not support nested inline mark up. So, for example, you can't have text that is both monospace and bold.